content/pt/docs/reference/glossary/cluster.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Cluster
+id: cluster
+date: 2020-08-03
+full_link: 
+short_description: >
+   Um conjunto de servidores de processamento, também chamados de "nodes", que executam aplicações containerizadas. Todo cluster possui ao menos um "worker node" (servidor de processamento).
```

Comment:
Ricardo, please, can you change the position like this: "worker node" (servidor de processamento) for this: servidor de processamento (worker node).

Reply:
ok, done